src/products/android-l10n/mobile_l10n_faqs.md

```diff
@@ -0,0 +1,139 @@
+
+We also have locales that have different codes between Pontoon and the repository for Android. For example:
+
+#### Firefox for Android (comprised of Fenix and Android-Components) and Focus for Android
+
+All localized files for Fenix, Android-Components and Focus live in the `mozilla-l10/android-l10n` repository, under [this folder](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile). As an example, [this](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile/focus-android/app/src/main/res) is where the French `strings.xml` translations for Focus would be found (under [values-fr](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile/focus-android/app/src/main/res/values-fr)).
+
+The locale folder will be displayed as \`values-{locale code}\`, and you will have an extra "r" for locales with hyphens (for example es-AR in [values-es-rAR](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile/fenix/app/src/main/res/values-es-rAR)).
+
+We also have some locale codes different for some folders: Hebrew is found under [values-iw](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile/fenix/app/src/main/res/values-iw) and Indonesian falls under [values-id](https://github.com/mozilla-l10n/android-l10n/tree/main/mozilla-mobile/fenix/app/src/main/res/values-in).
+
+#### Firefox for iOS
+
+Localized [files](https://github.com/mozilla-l10n/firefoxios-l10n). Each locale will have its own folder with one XLIFF file for the entire app.
+
+#### Focus for iOS
+
+Localized [files](https://github.com/mozilla-l10n/focusios-l10n). Each locale will have its own folder with one XLIFF file for the entire app.
+
+### Should I remove obsolete or unused strings?
+
+Yes. Removing obsolete or unused strings keeps source code clean and easier to maintain, while also improving the localization process. When deprecated strings remain in the project, localizers may spend valuable time translating or reviewing content that’s no longer in use — leading to wasted effort. By cleaning up unused strings, you ensure translators can focus on what truly matters: relevant, user-facing content. The links below will explain how to ensure you are correctly removing strings:
+
+* [Documentation](https://firefox-source-docs.mozilla.org/mobile/android/focus-android/Removing-strings.html#removing-strings) for Android
+* [Documentation](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#how-to-remove-a-string) for iOS
+
+Note that for both products, you must mark a string as "unused" rather than deleting it directly. This ensures the string stays available for versions where it’s still in use. The links above explain this process in further detail.
+
+### When should “Mozilla” or “Firefox” brand be hardcoded?
+
+We don’t usually hardcode brand names, but this can happen in some edge cases.If in doubt, reach out to the current localization mobile ePM or on the Slack #l10n channel.
```

Comment:
```suggestion
We don’t usually hardcode brand names, but this can happen in some edge cases. If in doubt, reach out to the current localization mobile ePM or on the Slack #l10n channel.
```